firebase/functions/index.js

```diff
@@ -129,10 +119,10 @@ exports.counter = functions.database.ref('/v2/results/{projectId}/{groupId}/{use
     return Promise.all(promises)
 })
 
-
-exports.counter_2 = functions.database.ref('/v2/users/{userId}/contributions/{projectId}/').onCreate((snapshot, context) => {
+// Counters to keep track of contributors and project contributions of Project and User.
+// Gets triggered when User contributes to new project.
+exports.contributionCounter = functions.database.ref('/v2/users/{userId}/contributions/{projectId}/').onCreate((snapshot, context) => {
```

Comment:
can we change this to `contributorCounter` instead?

Reply:
Renamed the function.